### PR TITLE
Fix a couple of cases

### PR DESCRIPTION
**Pull request by wez**
Details are in the individual commits, plus unit tests!

### CODE REVIEW

File: instant-xml-macros/src/de.rs [hunk 1]
```diff
@@ -75,10 +77,9 @@ fn deserialize_scalar_enum(
         impl #impl_generics FromXml<'xml> for #ident #ty_generics #where_clause {
             #[inline]
             fn matches(id: ::instant_xml::Id<'_>, field: Option<::instant_xml::Id<'_>>) -> bool {
-                match field {
-                    Some(field) => id == field,
-                    None => false,
-                }
+                let wanted = ::instant_xml::Id { ns: #default_namespace, name: id.name };
+
+                id == wanted
```
Comment (djc) [inline instant-xml-macros/src/de.rs:82]: Nit: maybe just spell this as `id == ::instant_xml::Id { .. }`?

File: instant-xml/tests/scalar-enum.rs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+#[derive(Debug, Eq, FromXml, PartialEq, ToXml)]
+#[xml(scalar, ns("URI", x = "URI"))]
+enum Bar {
+    A,
+    B,
+}
+
+#[derive(Debug, Eq, FromXml, PartialEq, ToXml)]
+#[xml(ns("OTHER", x = "URI"))]
+struct NSContainer {
```
Comment (djc) [inline instant-xml/tests/scalar-enum.rs:34]: Nit: please name it `NsContainer` per the API guidelines.

**Review by djc — COMMENTED** [review]
Thanks! Could you fix these nits (by squashing fixes into the existing commits)?

**Comment by wez** [comment]
Done!

**Comment by djc** [comment]
Would you like a release or are you comfortable with a git dependency pending further work?

**Comment by wez** [comment]
I'm happy with a local reference to my fork for the moment, so no need to rush with making a release!

**Review by djc — APPROVED** [review]

**Pull request merged**